include/bout/coordinates.hxx

```diff
@@ -114,8 +114,10 @@ public:
   /// Calculate differential geometry quantities from the metric tensor
   int geometry(bool recalculate_staggered = true,
       bool force_interpolate_from_centre = false);
-  int calcCovariant(); ///< Inverts contravatiant metric to get covariant
-  int calcContravariant(); ///< Invert covariant metric to get contravariant
+  /// Invert contravatiant metric to get covariant components
+  int calcCovariant(const std::string& region = "RGN_ALL");
+  /// Invert covariant metric to get contravariant components
+  int calcContravariant(const std::string& region = "RGN_ALL");
```

Comment:
If we use `RGN_NOCORNERS` maybe we should make that the default argument here.

Reply:
I'm leaving this as `RGN_ALL` as it keeps the previous behaviour and I think it's the least surprising as a default. Within the `Coordinates` ctors, we know we're setting the corners to NaN, but we don't know that in the general case.